### PR TITLE
Add Albania to the site

### PR DESCRIPTION
**Pull request by egyptianbman**
@wowprc's commit adds the initial Albanian translation. This commit adds
the various handling throughout the site to integrate the new translation.

### CODE REVIEW

File: translations/al.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+return array_merge(require_once __DIR__ . '/global.php', [
+    'Bitcoin 2.0:' => 'Bitcoin 2.0:',
```
Comment (egyptianbman) [inline translations/al.php:4]: This is just a starting point to show how translations should be done. For now, these will render in English.

File: al/index.php [hunk 2]
```diff
@@ -121,9 +121,9 @@ function gtag(){dataLayer.push(arguments);}
 <div class="container">
 <div class="row">
 <div class="col-md-12">
-<div class="heading-title-big"><br>It's your coin <br><span>BitcoinZ: The True Bitcoin 2.0</span></div>
-<div class="description-slide">Welcome to the decentralization</div>
-<div class="buttons-section"><a href="<?php echo $t['community-paper-link']; ?>" target="_blank" class="btn dark-btn large-btn"><?php echo $t['Community Paper']; ?></a><a href="https://connect.bitcoinz.global/" class="btn dark-btn large-btn" target="_blank">READ MORE</a><a href="https://bitcoinz.global/donations/" class="btn dark-btn large-btn" target="_blank">Donate</a><br><br><center><script type="text/javascript" src="/api.js"></script><div class="coinmarketcap-currency-widget" data-currency="bitcoinz" data-base="USD" data-secondary="" data-ticker="true" data-rank="true" data-marketcap="true" data-volume="true" data-stats="USD" data-statsticker="false"></div></center>
+<div class="heading-title-big"><br><?= $t['It\'s your coin']; ?><br><span><?= $t['BitcoinZ: The true Bitcoin 2.0']; ?></span></div>
```
Comment (renuzit) [inline al/index.php]: I would just do this here because it's easier to read:
```
$t["It's your coin"];
```

**Review by renuzit — APPROVED** [review]
Just one nit pick, but looks good!

File: al/index.php (hunk 2 above)
Comment (egyptianbman) [reply al/index.php]: Nit picks welcome, thanks!

**Pull request merged**